### PR TITLE
Add a comment for empty dist files

### PR DESCRIPTION
**Pull request by foolip**
This is to have something to search for, and another step towards always
having a dist file.

### CODE REVIEW

File: features/avif.yml.dist [hunk 1]
```diff
@@ -2,3 +2,5 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 {}
+
+# Nothing is generated, nothing to see here.
```
Comment (ddbeck) [inline features/avif.yml.dist:6]: What do you think of linking out to some documentation (you can just stub a heading for now, if you like) to handle the whole "why is it empty?" question?

Comment (foolip) [reply features/avif.yml.dist:6]: A link to https://github.com/web-platform-dx/web-features/blob/5f17741fd7c73119a8e25270ca2070d04d73cc0c/docs/CONTRIBUTING.md#feature-dist-files after it's landed, perhaps?

Comment (foolip) [reply features/avif.yml.dist:6]: Although, maybe it's better to link to the docs at the top of all dist files?

Comment (captainbrosset) [reply features/avif.yml.dist:6]: I agree with this. Linking to the docs on line 3 of all dist files (right after asking people not to edit the file by hand) would give them a friendly way to learn more, especially for folks who prefer to jump in the code first, and read docs later.

Comment (foolip) [reply features/avif.yml.dist:6]: Cool, I've sent https://github.com/web-platform-dx/web-features/pull/1345 as a draft, to land after https://github.com/web-platform-dx/web-features/pull/1330.

If the docs mention `{}` then I don't think this PR is needed at all.

**Pull request closed without merging**